### DISCUSSION TITLE
citing

### DISCUSSION THREAD

**Discussion by florent-abdelghafour** [TomSchimansky/CustomTkinter#1158] [Q&A] [answered]
Hi,
how to cite CustomTkinter in a paper?
I remember reading a reference, but I cannot find it anymore.

**Comment by IshanJ25** [comment]
like this?
```
Tom Schimansky, “CustomTkinter” A modern and customizable python UI-library based on Tkinter, 2023, https://github.com/TomSchimansky/CustomTkinter
```

**Reply by IshanJ25** [reply]
@TomSchimansky i think the author of the question is a little confused and has marked [this](https://github.com/TomSchimansky/CustomTkinter/discussions/1158#discussioncomment-5013010) as the answer.

Perhaps, could _[this one](https://github.com/TomSchimansky/CustomTkinter/discussions/1158#discussioncomment-5012802)_ (the actual answer) be marked instead?

**Comment by florent-abdelghafour** [comment, marked as answer]
Yes exactly! 
Thank you very much. 
----- Mail d’origine -----
De: Ishan Jindal ***@***.***>
À: TomSchimansky/CustomTkinter ***@***.***>
Cc: florent-abdelghafour ***@***.***>, Author ***@***.***>
Envoyé: Fri, 17 Feb 2023 18:25:32 +0100 (CET)
Objet: Re: [TomSchimansky/CustomTkinter] citing (Discussion #1158)

like this?

```

Tom Schimansky, “CustomTkinter” A modern and customizable python UI-library based on Tkinter, 2023, https://github.com/TomSchimansky/CustomTkinter

```





-- 

Reply to this email directly or view it on GitHub:

https://github.com/TomSchimansky/CustomTkinter/discussions/1158#discussioncomment-5012802

**Reply by IshanJ25** [reply]
if your problem was solved, could you mark my answer as "accepted"?
https://github.com/TomSchimansky/CustomTkinter/discussions/1158
